src/context/TodoContext.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, useState, useEffect } from 'react';
+
+const TodoContext = createContext();
```

Comment:
You don't really have to use `contextAPI` here. You are already using `localStorage` to store your tasks. Adding `contextAPI` on top of that doesn't have any benefit. Instead, it adds an extra layer of complexity. But I'm glad you implemented `contextAPI` here. Good Job. 👍🏼